packages/ceramic-cli/src/ceramic-daemon.ts

```diff
@@ -87,11 +100,41 @@ class CeramicDaemon {
     }
 
     const ceramic = await Ceramic.create(ipfs, ceramicConfig)
-    const idWallet = new IdentityWallet(async () => true, { seed })
+    const idWallet = new IdentityWallet(async () => true, { seed: cliConfig.seed })
     await ceramic.setDIDProvider(idWallet.get3idProvider())
     return new CeramicDaemon(ceramic, opts)
   }
 
+  /**
+   * Load/create CLI config
+   */
+  static async _loadCliConfig(): Promise<CliConfig> {
+    let exists
+    const fullCliConfigPath = path.join(DEFAULT_CLI_CONFIG_PATH, DEFAILT_CLI_CONFIG_FILE)
+    try {
+      await fs.access(fullCliConfigPath)
+      exists = true
+    } catch (e) {
+      exists = false
+    }
+
+    if (exists) {
+      const configJson = await fs.readFile(fullCliConfigPath, { encoding: 'utf8' })
+      return JSON.parse(configJson)
+    }
+
+    await fs.mkdir(DEFAULT_CLI_CONFIG_PATH, { recursive: true })
+
+    console.log('Generating identity wallet seed...')
+    const config: CliConfig = {
+      seed: '0x' + Buffer.from(nacl.randomBytes(32)).toString('hex') // create new seed
+    }
+
+    await fs.writeFile(fullCliConfigPath, JSON.stringify(config, null, 2))
+    console.log('Identity wallet seed generated')
```

Comment:
One console log is probably enough here? Let's keep the second one?

Reply:
makes sense! 👍